Rocket.Chat/Controllers/Auth/AuthTableViewController.swift

```diff
@@ -255,15 +266,18 @@ final class AuthTableViewController: BaseTableViewController {
 
         if isLoginServicesCollapsed {
             UIView.animate(withDuration: 0.5) {
-                self.collapsibleAuthSeparatorRow.showMoreButton.transform = CGAffineTransform(rotationAngle: CGFloat.pi)
+                self.collapsibleAuthSeparatorRow.showMoreButton.transform = CGAffineTransform(rotationAngle: .pi)
+                self.collapsibleAuthSeparatorRow.showMoreButton.accessibilityLabel =
+                    self.showMoreButtonAccessibilityLabel
             }
 
             tableView.deleteRows(at: extraLoginServiceIndexPaths, with: .automatic)
         } else {
             UIView.animate(withDuration: 0.5) {
-                self.collapsibleAuthSeparatorRow.showMoreButton.transform = CGAffineTransform(rotationAngle: CGFloat.pi * 2)
+                self.collapsibleAuthSeparatorRow.showMoreButton.transform = CGAffineTransform(rotationAngle: .pi * 2)
+                self.collapsibleAuthSeparatorRow.showMoreButton.accessibilityLabel =
```

Comment:
Why is this assignment inside animate [2]? And since it's the same in `else` as in `if` you should probably put it outside and reduce it to only one assignment.

Reply:
I have updated the PR. Thank you for the review!